src/main/java/org/spongepowered/api/world/generation/config/NoiseGeneratorConfig.java

```diff
@@ -86,29 +94,31 @@ static Builder builder() {
 
         Builder defaultFluid(BlockState fluid);
 
+        Builder surfaceRule(SurfaceRules rule);
+
         Builder seaLevel(int y);
 
+        Builder mobGeneration(boolean disableMobGeneration);
```

Comment:
Inversed boolean?

Reply:
it mirrors what vanilla has in its datapacks

Reply:
Perhaps but is still so confusing...I'd rather have the API be straightforward then graduate up this silly implementation detail.